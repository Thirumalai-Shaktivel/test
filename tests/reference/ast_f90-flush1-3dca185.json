{
    "basename": "ast_f90-flush1-3dca185",
    "cmd": "lfortran --show-ast-f90 --no-color {infile}",
    "infile": "tests/flush1.f90",
    "infile_hash": "d0c80e1628d5fabcef4bf73ec7d228e08c9d37462c1a3bd0f08481d8",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast_f90-flush1-3dca185.stdout",
    "stdout_hash": "e444b9cf414836f8eec3e1eb021123256dad2639304b2f386118c64c",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}