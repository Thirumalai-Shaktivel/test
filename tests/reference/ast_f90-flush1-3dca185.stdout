program flush1
flush(10, IOSTAT=n)
flush(20, IOMSG=n)
flush(ERR=label)
flush(30, UNIT=40)
flush(50)
end program flush1
